pyomo/contrib/mindtpy/config_options.py

```diff
@@ -237,15 +156,177 @@ def _get_GDPopt_config():
+                    "by expressions can cause problems. Exclude all duals "
+                    "smaller in absolute value than the following."
+    ))
+    CONFIG.declare("integer_tolerance", ConfigValue(
+        default=1E-5,
+        description="Tolerance on integral values."
+    ))
+    CONFIG.declare("constraint_tolerance", ConfigValue(
+        default=1E-6,
+        description="Tolerance on constraint satisfaction."
+    ))
+    CONFIG.declare("variable_tolerance", ConfigValue(
+        default=1E-8,
+        description="Tolerance on variable bounds."
+    ))
+    CONFIG.declare("zero_tolerance", ConfigValue(
+        default=1E-8,
+        description="Tolerance on variable equal to zero."
+    ))
+    CONFIG.declare("ecp_tolerance", ConfigValue(
+        default=None,
+        domain=PositiveFloat,
+        description="ECP tolerance",
+        doc="Feasibility tolerance used to determine the stopping criterion in"
+            "the ECP method. As long as nonlinear constraint are violated for "
+            "more than this tolerance, the method will keep iterating."
+    ))
+
+
+def _add_bound_configs(CONFIG):
+    CONFIG.declare("obj_bound", ConfigValue(
+        default=1E15,
+        domain=PositiveFloat,
+        description="Bound applied to the linearization of the objective function if master MILP is unbounded."
+    ))
+    CONFIG.declare("continuous_var_bound", ConfigValue(
+        default=1e10,
+        description="default bound added to unbounded continuous variables in nonlinear constraint if single tree is activated.",
+        domain=PositiveFloat
+    ))
+    CONFIG.declare("integer_var_bound", ConfigValue(
+        default=1e9,
+        description="default bound added to unbounded integral variables in nonlinear constraint if single tree is activated.",
+        domain=PositiveFloat
+    ))
+
+
+def _add_feas_pump_configs(CONFIG):
+    CONFIG.declare("fp_cutoffdecr", ConfigValue(
+        default=1E-1,
+        domain=PositiveFloat,
+        description="Additional relative decrement of cutoff value for the original objective function"
+    ))
+    CONFIG.declare("fp_iteration_limit", ConfigValue(
+        default=20,
+        domain=PositiveInt,
+        description="Feasibility pump iteration limit",
+        doc="Number of maximum iterations in the feasibility pump methods."
+    ))
+    # TODO: integrate this option
+    CONFIG.declare("fp_projcuts", ConfigValue(
         default=True,
-        description="add dual bound constraint to enforce the objective function should improve on the best found dual bound",
+        description="Whether to add cut derived from projection of MIP solution onto NLP feasible set",
         domain=bool
     ))
-    return CONFIG
+    CONFIG.declare("fp_transfercuts", ConfigValue(
+        default=True,
+        description="Whether to transfer cuts from the Feasibility Pump MIP to the DICOPT MIP (all except from the round in which the FP MIP became infeasible)",
```

Comment:
This is not DICOPT! Mention transferring cut to master MIP in selected 'strategy'